app/controllers/providers/check_provider_answers_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Providers
+  class CheckProviderAnswersController < BaseController
+    include Providers::ApplicationDependable
+
+    def index
+      @proceeding = @legal_aid_application.proceeding_types.first
```

Comment:
**[nit pick/not a blocker]** **For consistency**, given we're using the helper method `legal_aid_application` in the confirm action, we should probably use the same one here instead of using the instance variable directly. **Definitely not a blocker though.**

Reply:
👍